docs/sdk/SDK_API_WORKFLOW.md

```diff
@@ -0,0 +1,170 @@
+         * `MESH_CONN_TRANSPORT_ST2110_30` – SMPTE ST2110-30 Audio
+1. Apply the configuration:
+   * `mesh_apply_config_st2110()`
+
+## Configure connection – RDMA
+1. Define an RDMA configuration:
+   * `struct MeshConfig_RDMA`
+      * `.remote_ip_addr`
+      * `.remote_port`
+      * `.local_ip_addr`
+      * `.local_port`
+1. Apply the configuration:
+   * `mesh_apply_config_rdma()`
+
+## Configure payload – Video
+1. Define a video configuration:
+   * `struct MeshConfig_Video`
+      * `.width`
+      * `.height`
+      * `.fps`
+      * `.pixel_format` – `MESH_VIDEO_PIXEL_FORMAT_*`
+1. Apply the configuration:
+   * `mesh_apply_config_video()`
+
+## Configure payload – Audio
+1. Define an audio configuration:
+   * `struct MeshConfig_Audio`
+      * `.channels`
+      * `.sample_rate` – `MESH_AUDIO_SAMPLE_RATE_*`
+      * `.format` – `MESH_AUDIO_FORMAT_*`
+      * `.packet_time` – `MESH_AUDIO_PACKET_TIME_*`
+1. Apply the configuration:
+   * `mesh_apply_config_audio()`
+
+## Mesh DP SDK API internals – Connection configuration flowchart
+
+```mermaid
+flowchart LR
+   kind(.kind) --> sender[_Sender_]
+   kind --> receiver[_Receiver_]
+
+   conn_type(.conn_type) --> memif_t[_memif_]
+   conn_type --> st_t[_SMPTE ST2110-XX_]
+   conn_type --> rdma_t[_RDMA_]
+
+   conn(.conn) --> memif(.memif)
+      memif --> socket_path(".socket_path
+                             .interface_id")
+
+   conn --> st(.st2110) --> st_param(".remote_ip_addr
+                                      .remote_port
+                                      .local_ip_addr
+                                      .local_port")
+            st ---> transport(.transport)
+               transport --> st20["_SMPTE ST2110-20_ 
+                  Uncompressed Video"]
+               transport --> st22["_SMPTE ST2110-22_
+                  Constant Bit-Rate Compressed Video"]
+               transport --> st30["_SMPTE ST2110-30_
+                  Audio"]
+
+   conn --> rdma(.rdma) --> rdma_param(".remote_ip_addr
+                                        .remote_port
+                                        .local_ip_addr
+                                        .local_port")
+
+   payload_type(.payload_type) --> video_t(_Video_)
+   payload_type --> audio_t(_Audio_)
+
+   payload(.payload) --> video(.video) --> videoparam(".width
+                                                       .height
+                                                       .fps
+                                                       .pixel_format")
+   payload --> audio(.audio) --> audioparam(".channels
+                                             .sample_rate
+                                             .format
+                                             .packet_time")
+```
+
+## Mesh DP SDK API internals – Connection configuration mindmap
+```mermaid
+mindmap
+)__MeshConnection__ configuration(
+   **.kind**
```

Comment:
".kind" branch is drawn in yellow. (It is ok.) And it overwrites violet ".conn" branch. Maybe there is some way to enforce lack of overwriting?

Reply:
Unfortunately, there is no option to control the rendering process of Mermaid.

Reply:
Maybe I'll remove the mind map from this document since it doesn't bring more clarity to the configuration.

Reply:
it is quite clear. I just suggested to do it even more clear. Keep it that way.